### PR TITLE
[FIX] Change state in the "picking state"

### PR DESCRIPTION
**Pull request by JoelZilli**

### CODE REVIEW

File: stock_ux/models/stock_picking.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def action_done(self):
             # se actualice
             # antes lo haciamo en do_new_transfer, pero como algunas
             # veces se llama este metodo sin pasar por do_new_transfer
-            if picking.state not in ['draft', 'confirmed', 'assigned']:
+            if picking.state not in ['done', 'cancel']:
```
Comment (nicomacr) [inline stock_ux/models/stock_picking.py]: here the condition must be the opposite `in` instead `not in`

**Review by nicomacr — CHANGES_REQUESTED** [review]

**Comment by nicomacr** [comment]
@roboadhoc r+

**Comment by nicomacr** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
Sorry, I didn't know about this PR and had to retrieve its information, you may have to re-approve it.

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/stock/pull/306).

**Comment by roboadhoc** [comment]
I'm sorry, @nicomacr. This PR is already reviewed, reviewing it again is useless.

**Comment by nicomacr** [comment]
@roboadhoc r+

**Pull request closed without merging**